satpy/etc/readers/lsasaf-geo.yaml

```diff
@@ -0,0 +1,44 @@
+reader:
+  name: lsasaf-geo
```

Comment:
Is the name of the software "LSASAF GEO"? If not, then I'd say an underscore in the reader name would be more consistent. Or even "lsasaf" if there is no LEO version.